sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-assessment-workflow-enqueuer-queue.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO
+  sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+  (
+    'assessment-workflow-enqueuer',
+    '{
+        "dashboardReadonly": false,
```

Comment:
Why we are not having a retry here, just for my understanding.

Reply:
Because it runs every 30 seconds 😉 
![image](https://github.com/bcgov/SIMS/assets/61259237/800b2857-175d-4075-bb1e-9ab952e93eb4)